ara_plumes/regressions.py

```diff
@@ -120,75 +121,57 @@ def do_parametric_regression(
 
 
 def edge_regression(
```

Comment:
IIUC, this is no longer an "edge" regression, but instead a generic regression dispatcher?  Like it could be used for any data?

Reply:
yes! it can be used for any multivariate data.